Documentation/contributing/release/backports.rst

```diff
@@ -200,12 +200,12 @@ The comment must not contain any other characters.
 After the backports are merged
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After the backport PR is merged, mark all backported PRs with
-``backport-done/X.Y`` label and clear the ``backport-pending/X.Y`` label(s). If
-the backport pull request description was generated using the scripts above,
-then the full command is listed in the pull request description.
+After the backport PR is merged, if the person who merged the PR didn't take
+care of it already, mark all backported PRs with ``backport-done/X.Y`` label
+and clear the ``backport-pending/X.Y`` label(s). If the backport pull request
+description was generated using the scripts above, then the full command is
+listed in the pull request description.
```

Comment:
Note that release notes for new patch releases rely on this section so it is required to be there (with the ```upstream-prs ... syntax)

Reply:
I've noticed a few times now that some backport PRs are missing the end ``` for the `upstream-prs` block, which would cause our tooling to crash. I'm not sure if we have a bug in the backporting scripts, or if people are writing this block by hand, but I +1 adding this to the docs at least.

Reply:
@joestringer Aah, you were suggesting we add a note on the `upstream-prs` block. I did not get that :-)

I'll add a `.. note::`. We should probably ping the folks who did that nonetheless, because it's unlikely they will notice the small documentation update...